src/bitcoin-cli.cpp

```diff
@@ -245,7 +245,7 @@ class BaseRequestHandler
 class AddrinfoRequestHandler : public BaseRequestHandler
 {
 private:
-    static constexpr std::array m_networks{"ipv4", "ipv6", "torv2", "torv3", "i2p"};
+    static constexpr std::array m_networks{"ipv4", "ipv6", "onion", "i2p"};
```

Comment:
nit: maybe we can write torv3/v3 in bracket for few months

```suggestion
    static constexpr std::array m_networks{"ipv4", "ipv6", "onion(v3)", "i2p"};

Reply:
That wouldn't work (but could be done by keeping and modifying the conditional that this patch removes). By the time 23.0 is released in 2022, my thought was that it should be widely known that only v3 onions are supported by Bitcoin Core and current releases of Tor, and "onion" is both simpler to implement and the same as the other RPCs.